### DISCUSSION TITLE
Is application/json;odata.metadata=none Supported and can you set a default

### DISCUSSION THREAD

**Discussion by p6345uk** [OData/AspNetCoreOData#958] [Q&A]
I believe that adding an Accept header of application/json;odata.metadata=none

Should omit the "@odata.context" as per the OData spec however I cannot seem to get it to work.

Also does anyone know if there a way to set the default to none from minimal?

**Comment by p6345uk** [comment]
Sounds like its not a thing :(

**Comment by habbes** [comment]
Hi @p6345uk You should use the `Prefer` header or `$format` query option to specify preferences.

Using `Prefer` header:
```
Prefer: odata.metadata=none
```

Using `$format` query option:

```
GET myservice/endpoint?$format=application/json;odata.metadata=none
```

Let us know if this works. If it doesn't work, consider filing an issue.
